src/panel/service/ServicePanelDesktop.jsx

```diff
@@ -1,24 +1,26 @@
 /* global _ */
-import React from 'react';
+import React, { Fragment } from 'react';
 import Panel from 'src/components/ui/Panel';
 import CategoryList from 'src/components/CategoryList';
 import Action from 'src/components/ui/MainActionButton';
 import nconf from '@qwant/nconf-getter';
 
+const directionConf = nconf.get().direction;
+
 class ServicePanelDesktop extends React.Component {
   render() {
     return <Panel
       className="service_panel"
-      title={<div className="u-text--smallTitle u-center">
-        {_('Qwant Maps services', 'service panel')}
-      </div>}
       white
     >
-      <CategoryList className="service_panel__categories" />
+      <div className="service_panel__actions">
 
-      <hr/>
+        <h3 className="u-text--smallTitle u-center">{_('Qwant Maps Services', 'service panel')}</h3>
+
+        <CategoryList className="service_panel__categories" />
+
+        <hr/>
```

Comment:
can we use utils class like `.u-mr-8` for instance instead of hr ?

Reply:
this class only adds a 8px margin bottom, while hr includes a horizontal line

Reply:
but I added a missing container around the action buttons so the artificial hr's margin is no longer necessary ;)